blocks/query/edit.tsx

```diff
@@ -44,6 +44,7 @@ interface Window {
   wpCurateQueryBlock: {
     allowedPostTypes: Array<string>;
     allowedTaxonomies: Array<string>;
+    useParsely: string,
```

Comment:
Is this really to determine whether Parse.ly support is _available_, rather than whether the query should use Parse.ly to pull its data (vs. a term-based query)? If so, I'd recommend naming this key something like `parselyAvailable`